_posts/2016-08-02-should-i-use-a-spy-a-double-or-an-instance-double.markdown

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: "Should I use a spy, a double, or an instance_double?"
```

Comment:
@etagwerker I didn't edit the title as the idea was not to focus on their differences, but rather, why use an `instance_double`, or even a `double`, over `spy`. If you have other title ideas, let me know!